cypress/e2e/capture.cy.ts

```diff
@@ -25,6 +25,20 @@ describe('Event capture', () => {
         cy.phCaptures().should('include', 'custom-event')
     })
 
+    it('captures pageviews with current url', () => {
+        start({})
+
+        cy.get('[data-cy-custom-event-button]').click()
```

Comment:
i don't _think_ the nuxt client affects these cypress pages...

i believe you can get to the page's posthog instance, i'd be tempted to call `cy.posthog.page('https://not-the-page-i-loaded;)` (assuming that's how you get to the posthog intance)

and then assert that URL has been captured on a pageview

a little more explicit and certain that way

Reply:
@pauldambra I've made the changes. Please have a look.